app/views/components/_guest_nav_drawer.html.erb

```diff
@@ -1,2 +1,3 @@
-<!-- <a href="/login" class="button&#45;login usa&#45;button usa&#45;button&#45;outline&#45;inverse">Login</a> -->
-<%= link_to 'Login', login_path, class: 'usa-button usa-button-outline-inverse' %>
+<% if request.path != '/login' %>
```

Comment:
yeah I wasn't sure how else to do it. will probably be thrown out but I am open to ideas on a better way!